internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -634,10 +634,12 @@ func flattenAdvancedReplicationSpecRegionConfigSpec(apiObject *admin.DedicatedHa
 	if len(tfMapObjects) > 0 {
 		tfMapObject := tfMapObjects[0].(map[string]any)
 
-		if providerName == "AWS" {
+		if providerName == constant.AWS || providerName == constant.AZURE {
```

Comment:
Could an additional check in TestAccClusterAdvancedCluster_multicloudSharded capture this change (for the replication spec using azure)?

Reply:
done, thx